src/pages/Player.vue

```diff
@@ -224,6 +250,12 @@ export default {
       // styling class for the minimize button
       maximizeButtonClass:
         "bg-primary hover:bg-primary-hover p-1 pl-4 pr-4 sm:p-2 sm:pl-6 sm:pr-6 lg:p-4 lg:pl-6 lg:pr-6 rounded-md shadow-xl disabled:opacity-50 disabled:pointer-events-none",
+      numOfCorrect: 0, // number of correctly answered questions (valid for MCQ only)
+      numOfWrong: 0, // number of wrongly answered questions (valid for MCQ only)
```

Comment:
should we remove the mcq part in detail? will change for subjective now + for other new question types will have to remember to edit this